search-app/src/cmr/search/services/query_execution/granule_counts_results_feature.clj

```diff
@@ -83,24 +83,26 @@
 
 (defn- extract-spatial-operator
   "Determine whether spatial conditions should be ORed or ANDed together"
-  [query]
-  (let [conditions (get-in query [:condition :conditions])
-        condition-is-shape? (->> conditions
-                                 (map keys)
-                                 flatten
-                                 (filter #(= % :shape))
-                                 seq)
-        parent-group-operator (get-in query [:condition :operation])]
-    (when condition-is-shape?
-      parent-group-operator)))
+  [context]
+  (if (empty? (:query-string context))
+      ; If there's no query string, we can safely assume that there is no
+      ; options[spatial][or] parameter being passed
+      :and
+      (let [query-string (:query-string context)
+            parameters (string/split query-string #"\?|&")
+            spatial-param-regex #"(options%5Bspatial%5D%5Bor%5D=true|options\[spatial\]\[or\]=true)"
+            spatial-or-option? (some #(re-matches spatial-param-regex %) parameters)]
+        (if spatial-or-option?
+          :or
+          :and))))
 
 (defn- combine-spatial-and-non-spatial-conditions
   "Helper function. If spatial conitions are ORed, combine them with the rest of
   the conditions appropriately."
   [spatial-conditions temporal-conditions collection-ids]
   (let [collection-id-conditions (q/string-conditions :collection-concept-id collection-ids true)
         ored-spatial-conditions (gc/or-conds spatial-conditions)]
-    (if (seq collection-ids)
+    (if (seq? collection-ids)
```

Comment:
Why do we make this change?

Reply:
I was testing something and forgot to reverse it. `seq` vs `seq?` doesn't seem to make a difference in this case, but I'm still going to change it back